src/panels/mailbox/ha-panel-mailbox.js

```diff
@@ -129,7 +129,7 @@ class HaPanelMailbox extends LocalizeMixin(PolymerElement) {
       </div>
     </app-header-layout>
 
-    <paper-dialog with-backdrop id="mp3dialog" on-iron-overlay-closed="_mp3Closed">
+    <paper-dialog id="mp3dialog" on-iron-overlay-closed="_mp3Closed">
```

Comment:
The dialog should be tweaked and use the dialog manager to properly handle overlaying all content.  Example: https://github.com/home-assistant/home-assistant-polymer/blob/18a151c8e8f3c0b3d14e75b98363852e3b776da6/src/panels/config/config-entries/ha-config-entries.js#L127-L134